+euphonic/install_python_modules.m

```diff
@@ -0,0 +1,21 @@
+function install_python_modules()
+    try
+        req_mods = required_modules;
+        verify_python_modules(req_mods{:});
+    catch ME
+        % Checks we have the right Python version (>=3.6)
+        if ~strcmp(ME.identifier, 'euphonic_horace_interface:verify_python_modules:pythonVersion')
+            rethrow(ME);
+        end
+    end
```

Comment:
I'm not sure about this logic, if we don't have the right python version we **do** want to rethrow the error right? Because we can't install the correct Python version for the user.

We want to continue running the script if we get the errors `euphonic_horace_interface:verify_python_modules:modVersionUnavailable` (euphonic/euphonic-horace-interface not installed) or `euphonic_horace_interface:verify_python_modules:modIncompatibleVersion` (installed but incorrect version) right?

Reply:
Oh yes, I wasn't thinking straight on this... is should be `strcmp` not `~strcmp`....